### PR TITLE
Address feedback recieved on #74

### PR DESCRIPTION
**Pull request by alshakero**
This is a follow-up to https://github.com/Automattic/mShots/pull/74 with the same testing steps.

### CODE REVIEW

File: public_html/class-mshots.php [hunk 1]
```diff
@@ -348,7 +346,7 @@ protected function resolve_filename( $snap_url ) {
 			if ( $this->scale_factor != self::SCALE_FACTOR_DEFAULT ) {
 				$suffix .= "_{$this->scale_factor}x";
 			}
-			
+
 			$extension = $this->format === 'jpeg' ? '.jpg' : '.png';
```
Comment (aidvu) [inline public_html/class-mshots.php:350]: Same as above for default.

Comment (alshakero) [reply public_html/class-mshots.php]: Addressed and tested 👍🏼

File: public_html/class-mshots.php [hunk 2]
```diff
@@ -23,6 +23,10 @@ class mShots {
 		const SCREEN_MAX_H = 3600;
 		const SCALE_FACTOR_VALUES = [1, 2];
 		const SCALE_FACTOR_DEFAULT = 1;
+		const FORMAT_TO_EXTENSION = [
```
Comment (aidvu) [inline public_html/class-mshots.php:26]: Could you make a block of the 3 new constants below the others.

So, `DEFAULT_FORMAT`, `ALLOWED_FORMATS`, `FORMAT_TO_EXTENSION`.

File: lib/master.js [hunk 3]
```diff
@@ -9,23 +9,20 @@ const fs      = require( 'fs' );
...
-const EXIT_ERROR = 3;
-const EXIT_FORCED = 143;
-
-const SUCCESS = 0;
-const HOST_NO_DNS = 1;
-const HOST_BLACKLISTED = 2;
-const HOST_INVALIDSCHEMA = 3;
-const HOST_INVALID = 4;
-const GENERAL_ERROR = 5;
-const FORMAT_ERROR = 6;
```
Comment (alshakero) [inline lib/master.js:28]: `FORMAT_ERROR` was never used

File: lib/master.js [hunk 4]
```diff
@@ -9,23 +9,20 @@ const fs      = require( 'fs' );
...
 const logger  = _log4js.getLogger( 'flog' );
 
-const MAX_WORKERS = 50;
-const MAX_NO_RESPONSE_COUNT = 3;
-const MAX_QUEUE_PROCESS_WAIT_MSECS = 45000;
-const STATS_UPDATE_INTERVAL_MSECS = 15000;
-
-const EXIT_MAXRAMUSAGE = 1;
-const EXIT_COMMANDED = 2;
-const EXIT_ERROR = 3;
```
Comment (alshakero) [inline lib/master.js:19]: `EXIT_ERROR` was never used.

File: lib/snapshot.js [hunk 5]
```diff
@@ -7,24 +7,24 @@ const _fs                = require( 'fs' );
...
-const SUCCESS             = 0;
-const HOST_NO_DNS         = 1;
-const HOST_BLACKLISTED    = 2;
-const HOST_IP_BLACKLISTED = 3;
-const HOST_INVALIDSCHEMA  = 4;
-const HOST_INVALID        = 5;
-const GENERAL_ERROR       = 6;
-
-const EXIT_UNRESPONSIVE   = 4;
-const EXIT_ERROR_LIMIT    = 5;
```
Comment (alshakero) [inline lib/snapshot.js:26]: `EXIT_ERROR_LIMIT` was never used.

File: public_html/class-mshots.php [hunk 6]
```diff
@@ -257,7 +261,6 @@ private function image_resize_and_output( $image_filename, $format ) {
 						} else {
 							imagepng( $image );
 						}
-						return;
```
Comment (aidvu) [inline public_html/class-mshots.php:260]: This one stays. :) You added the other one by mistake.

**Comment by aidvu** [comment]
There's still some uses of `jpg/jpeg` hardcoded strings. Will change those next week.

**Pull request merged**